operator-pipeline-images/Dockerfile

```diff
@@ -55,10 +56,9 @@ RUN curl -LO https://github.com/operator-framework/operator-registry/releases/do
     mv operator-sdk_linux_${ARCH} /usr/local/bin/operator-sdk
 
 RUN useradd -lms /bin/bash -u "${USER_UID}" user
-
 RUN useradd -lu "${PODMAN_USER_UID}" podman; \
-    echo podman:10000:5000 > /etc/subuid; \
-    echo podman:10000:5000 > /etc/subgid;
+    echo podman:10000:5000 >> /etc/subuid; \
+    echo podman:10000:5000 >> /etc/subgid;
```

Comment:
We can remove those since the version of `useradd` shipped with fedora 38 already generates subuid and subgid mappings when creating new users.

Reply:
Fixed